code-challenges.js

```diff
@@ -29,23 +76,56 @@ var nums1 = [3, 56, 90, -8, 0, 23, 6]
 var nums2 = [109, 5, 9, -59, 8, 24]
 // Expected output: [-59, 109]
 
+//create a function that takes in an array
+const minMax = (array) => {
+    // Return the minimum and maximum numbers from the array. Use Array.from to convert the 2 numbers into an array that we can return.
+    return Array.from([(Math.min(...array)), (Math.max(...array))])
+}
 
-
-
+// console.log(minMax(nums1));
+// console.log(minMax(nums2));
 
 // --------------------4) Create a function that takes in a string and returns a string with every other letter capitalized.
 
+// var testString1 = "albatross"
 var testString1 = "albatross"
 // Expected output: "aLbAtRoSs"
 var testString2 = "jabberwocky"
 // Expected output: "jAbBeRwOcKy"
 
-
-
-
+//create a function that takes in a string
+const funkyCaps = (string) => {
+    //convert the string to an array using spread operator, example: ["a", "l", "b", "a", "t", "r", "o", "s", "s"]
+    let arr = [...string]
+    //create a new array, map over arr. 
+    let newArr = arr.map((value, index) => {
+        // if index is even, change to lowercase.
+        if (index%2 === 0){
+            return string[index].toLowerCase()
+        // else change to uppercase.
+        } else{
+            return string[index].toUpperCase()
+        }
+    })
+    //return newArr, join back into a string and remove all the commas
+    return newArr.join().replace(/,/g, '')
```

Comment:
oOoh, some RegEx I see! Super excited to see this, but next time let's comment what its doing here exactly.

Looking great!